.setup/pip/dev_requirements.txt

```diff
@@ -1,9 +1,9 @@
-# Devlopment Requirements
+# Development Requirements
 
 # Linters
 pylint==3.0.2
 flake8-bugbear==23.12.2
 flake8==6.1.0
 
 # Code Formatter
-black==24.3.0
+black==23.11.0
```

Comment:
Is black being used? ( There was a conversation a while ago that I thought it was decided that we weren't going to use black, because it does some formatting that isn't what Submitty's standards are)

Reply:
While Black is not necessarily ideal, and isn't being used in the CI yet, I believe @jason490 used it to automatically reformat a bunch of the Python files in an attempt to reduce the number of linting warnings.  It might be worth revisiting whether requiring Black formatting would be beneficial in the future.

In this case, it looks like the version got changed during a bad merge resolution, so I'll revert it.